Extension/src/LanguageServer/ui.ts

```diff
@@ -426,12 +433,20 @@ export class LanguageStatusUI {
     private showConfigureIntelliSenseButton: boolean = false;
     private configureIntelliSenseTimeout?: NodeJS.Timeout;
 
-    public async ShowConfigureIntelliSenseButton(show: boolean, client?: Client): Promise<void> {
-        if (!await telemetry.showStatusBarIntelliSenseButton() || client !== this.currentClient) {
+    public async ShowConfigureIntelliSenseButton(show: boolean, client?: Client, configurationType?: ConfigurationType, sender?: string): Promise<void> {
```

Comment:
Change the type of `sender` to an enum, similar to `enum ConfigurationType` to avoid typos in re-usage of the same senders.

Reply:
We don't do this with the other APIs accepting `sender`.  Maybe a const string would be helpful, but I don't think the intent is to keep this telemetry forever.

Reply:
If string is commonly used, then I suggest to create a local `const string` where `ShowConfigureIntelliSenseButton` is called multiple times with the same sender.

Reply:
done